source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -39,21 +39,98 @@ You can create a Prometheus service within your PaaS space and allow it to bind
 Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
 
-### App route
+### App route configuration
 
 Whether or not you're using a custom domain the Prometheus service broker will only scrape your app's first route.
 
 For example, some apps may have multiple routes especially if they use custom domains such as `custom.domain.gov.uk, an-interesting-app.cloudapps.digital`. Only the first route will be picked up.
 
-If there are no routes to your app the Prometheus service will default the route to `<app-name>.cloudapps.digital`.
+If there are no routes to your app the Prometheus service will default the route to:
+
+`<app-name>.cloudapps.digital`.
 
 Because changes to the first route will only be picked up within 5 minutes it could create gaps to metrics during route changes if you're using a zero downtime plugin such as [autopilot][].
 
+## Configure container metrics
+
+[Cloud Foundry][] provides time-series data (metrics), for your PaaS apps.
+
+Use the metrics exporter app to send Cloud Foundry metrics data to a monitoring system of your choice to store and view it. You'll need to deploy the metrics exporter as an app on PaaS before you can use it.
+
+Currently suppprted metrics are:
+
+* CPU
+* RAM
+* disk usage data
+* app crashes
+* app requests
+* app response times
+
+### Set up the metrics exporter app
+
+Before you setting up the metrics exporter app, you'll need a live Cloud Foundry account assigned to the spaces you want to receive metrics on.
+
+Your new account should be separate to your primary Cloud Foundry account and use the [`SpaceAuditor`][] role beause it can view app data without modifying it.
+
+To set up the metrics exporter app:
+
+1. Clone the [paas-metric-exporter][] GitHub repository.
+2. [Push the metrics exporter app](https://docs.cloud.service.gov.uk/#deployment-overview) to Cloud Foundry (without starting the app) by running
+ * `cf push --no-start metric-exporter`
+3. Set the following mandatory environment variables in the metrics exporter app using
+  * `cf set-env metric-exporter NAME VALUE`
+
+	Use the `cf set-env` command for these mandatory variables, this will keep secure the secret information contained in them.
+
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|Use `https://api.cloud.service.gov.uk`|
+	|`USERNAME`|Cloud Foundry user|
+	|`PASSWORD`|Cloud Foundry password|
+	|`ENABLE_STATSD`|false|
+	|`ENABLE_PROMETHEUS`|true|
+
+	You could also set environment variables by amending the manifest file for optional environment variables that do not contain secret information. Read [paas-metric-exporter][] GitHub repository for more information.
+
+4. Run `cf start metric-exporter` to start your app
+1. Check you're generating Prometheus metrics at the metrics endpoint
+    - `https://<your-app-url>/metrics`
+5. Bind your app to the Prometheus service
+  - `cf bind-service <app-name> <service-instance-name>`
+
+### IP whitelist your app
+
```

Comment:
We should have an intro here explaining why we have the section and why people should do this. It's probably self-explanatory for the target audience but it'll help those with less subject knowledge.

Reply:
Will do